groovy/postInit/chains/FermentingChain.groovy

```diff
@@ -77,7 +86,7 @@ FERMENTER.recipeBuilder()
 FERMENTER.recipeBuilder()
```

Comment:
Biologically impossible for microorganisms to survive in anhydrous ethanol. Revert this.